src/test/javascript/spec/component/about-us/about-us.component.spec.ts

```diff
@@ -47,4 +48,29 @@ describe('AboutUsComponent', () => {
             expect(getStaticJsonFromArtemisServerStub).toHaveBeenCalledTimes(1);
         });
     }));
+
+    it('load and display contributors', fakeAsync(() => {
+        const staticContentService = TestBed.inject(StaticContentService);
+        const profileService = TestBed.inject(ProfileService);
+
+        const fullName = 'Full Name';
+        const photoDirectory = 'Photo Directory';
+        const role = 'ADMIN';
+        const website = 'www.website.de';
+
+        const contributors = [new ContributorModel(fullName, photoDirectory, role, website)];
+
+        const getStaticJsonFromArtemisServerStub = jest.spyOn(staticContentService, 'getStaticJsonFromArtemisServer').mockReturnValue(of(new AboutUsModel([], contributors)));
+        const getProfileInfoSub = jest.spyOn(profileService, 'getProfileInfo');
+        getProfileInfoSub.mockReturnValue(
+            new BehaviorSubject<ProfileInfo>({ inProduction: false, sshCloneURLTemplate: 'ssh://git@testserver.com:1234/' } as ProfileInfo).asObservable(),
```

Comment:
```suggestion
            of({ inProduction: false, sshCloneURLTemplate: 'ssh://git@testserver.com:1234/' } as ProfileInfo)
```
You just want an observable, right? Or is there a reson to use a subject?

Reply:
Yes exactly. Nice trick 👍🏻